src/press/views.py

```diff
@@ -42,12 +47,13 @@ def index(request):
 
     # call all registered plugin block hooks to get relevant contexts
     for hook in settings.PLUGIN_HOOKS.get('yield_homepage_element_context', []):
-        hook_module = plugin_loader.import_module(hook.get('module'))
-        function = getattr(hook_module, hook.get('function'))
-        element_context = function(request, homepage_elements)
+        if hook.get('name') in homepage_element_names:
```

Comment:
This hides the problem rather than solving it, we should at least log a warning when 'name' is not in homepage_element_names alerting of the bad configuration